capstone/fitness/src/main/java/com/google/sps/servlets/PaginationServlet.java

```diff
@@ -99,6 +82,6 @@ else if(movePage.equals("next")) {
         }
       }
     }
-    return new Metadata(count, page, filterData);
+    return new Metadata(count, filterData);
```

Comment:
Consider renaming this class since it doesn't really handle pagination anymore.

Reply:
done; renamed DisplayParameters